src/PhiremockFactory.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mcustiel\Phiremock\Client;
+
+use Http\Discovery\Psr17FactoryDiscovery;
+use Mcustiel\Phiremock\Factory as Base;
+use Psr\Http\Message\RequestFactoryInterface;
+use Psr\Http\Message\StreamFactoryInterface;
+
+class PhiremockFactory extends Base
+{
+    public function findRequestFactoryInterface(): RequestFactoryInterface
+    {
+        if (!class_exists('\Http\Discovery\Psr17FactoryDiscovery', true)) {
```

Comment:
If we require php-http/factory to work. Then it should be in the "require" section, not in the "require-dev"